src/tcpconns.c

```diff
@@ -171,6 +176,24 @@ struct tcpi_field_selector {
 static struct tcpi_field_selector *tcpi_fields_to_report = NULL;
 static size_t num_tcpi_fields_to_report = 0;
 static size_t tcpi_fields_to_report_size = 0;
+static size_t num_tcpi_counter_fields = 0;
+
+static uint64_t counter_max_delta = 1000000;
+
+/* Cache for computing deltas of DELTA values. */
+static cdtime_t counter_cache_timeout = DOUBLE_TO_CDTIME_T(120.0);
+struct counter_cache_entry_s {
+  /* Cache entry holds num_tcpi_counter_fields values -- one 64-bit value
+   * for every COUNTER entry in order in tcpi_fields_to_report, This is
+   * later used to compute a new value including wrap-around. */
+  uint64_t *values;
+
+  cdtime_t last_update_time;
+};
+typedef struct counter_cache_entry_s *counter_cache_entry_t;
```

Comment:
It's a bit confusing (I expected counter_cache_entry_t to be counter_cache_entry_s, not counter_cache_entry_s*)